src/axom/primal/geometry/NURBSPatch.hpp

```diff
@@ -1142,7 +1296,8 @@ class NURBSPatch
   {
     return (lhs.m_controlPoints == rhs.m_controlPoints) &&
       (lhs.m_weights == rhs.m_weights) && (lhs.m_knotvec_u == rhs.m_knotvec_u) &&
-      (lhs.m_knotvec_v == rhs.m_knotvec_v);
+      (lhs.m_knotvec_v == rhs.m_knotvec_v) &&
+      (lhs.m_trimmingCurves == rhs.m_trimmingCurves);
```

Comment:
Does this also need to test the `m_isTrimmed`?
It seems that patches w/o curves but w/ different `m_isTrimmed` values should not be considered identical

Reply:
Good catch!